src/components/Modal/Modal.tsx

```diff
@@ -59,13 +55,40 @@ export const Modal = ({
     return () => {
       document.removeEventListener("keydown", keyDown);
     };
-  }, [close]);
+  });
+
+  const modalRef = React.useRef();
+  const handleTabKey = (e) => {
+    const focusableModalElements = modalRef.current.querySelectorAll(
```

Comment:
It's possible that this returns undefined so you'll need to handle that eventuality. I'm surprised the TS compiler didn't complain about this. You might want to check that you have error reporting enabled in your editor.